python/src/aiconfig/editor/client/src/components/prompt/model_settings/ModelSettingsConfigRenderer.tsx

```diff
@@ -1,4 +1,4 @@
-import { JSONObject } from "aiconfig/dist/common";
+import { JSONObject } from "aiconfig";
```

Comment:
Ah, I had these same changes in https://github.com/lastmile-ai/aiconfig/pull/643 , plus using the import in the parameters renderer. Not an issue, I can rebase after this lands